bot/src/plugins/github_issues.cr

```diff
@@ -1,8 +1,19 @@
 require "framework/plugin"
+require "http/client"
+require "json"
 
 class GithubIssues
   include Framework::Plugin
 
+  def get_issue msg, issue_id
+    r = HTTP::Client.get("https://api.github.com/repos/#{config.repositories[msg.channel.name]}/issues/#{issue_id}", headers: HTTP::Headers{"Accept" => ["application/vnd.github.v3+json"], "User-Agent" => ["cbot_issue_fetch"]})
+    if r.status_code == 200
+      j = JSON::Parser.new(r.body).parse
+      j = j as Hash
+      return "#{j["html_url"]} (#{j["title"]})"
+    end
+  end
```

Comment:
I generally like to order code so that calling methods are above called ones, and also this one can be private :)

```cr
def a
  b
end

private def b
end
```